frontend/public/components/_nav.scss

```diff
@@ -52,27 +53,28 @@ $active-border: 3px;
   -webkit-overflow-scrolling: touch;
 }
 
+.navigation-container--open {
+  background-color: $color-pf-black-800;
+}
+
 .navigation-container__list {
   margin: -10px 0 10px 0;
   overflow-y: hidden;
   padding-left: 0;
   transition: max-height 0.25s;
 }
 
-.navigation-container__section, .navigation-container__section__separator {
+.navigation-container__section {
   border-bottom: 1px solid $color-os-nav-item-seperator;
```

Comment:
The var name doesn't align with the class name.  `$color-os-nav-item-seperator` should go with `.navigation-container__section__separator`.  Adjust accordingly.